Microsoft.Toolkit/Observables/ObservableGroup.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+
+namespace Microsoft.Toolkit.Observables.Collections
```

Comment:
This seems like a very specific namespace, would it make more sense to just create a `Microsoft.Toolkit.Collections` and have these in there and give us room for the future? They already have `Observable` in their names, so I think it'd be clear.  We could move the `IIncrementalLoadingCollection` interface into this folder as well. Thoughts?

Reply:
I've moved all my code in the `Microsoft.Toolkit.Collections` namespace.